.dockerignore

```diff
@@ -0,0 +1,14 @@
+/data
+/build
+/images
+/LICENSE
+/AUTHORS
+/notebooks
+/README.md
+/docker-compose.yml
```

Comment:
I suggest dropping `docker-compose.yml` and `Dockerfile` as we may need later for debugging / other reasons.

Reply:
These files are just excluded from being sent to the build context when building the image. According to [this](https://docs.docker.com/engine/reference/builder/#dockerignore-file) the Dockerfile and .dockerignore are sent to the Docker daemon anyway but not unnecessarily copied when using ADD or COPY statements. As the whole repo directory is mounted as a volume anyway when starting a container they can still be accessed from within the container.